migrate_pim/migrate_to_saas.rst

```diff
@@ -44,11 +44,83 @@ compare your PIM version (*ex:* ``7.0.123``) with the latest version of Akeneo P
 
 .. tip::
 
-    To check if you need to upgrade, you can scroll to the bottom of your PIM's UI — 
+    To check if you need to upgrade, you can scroll to the bottom of your PIM's UI —
     if the PIM can be upgraded, you will see a message like ``Version: EE 7.0.20 Buckwheat | New patch available: v7.0.29``.
 
 Guides to update the PIM to latest patch can be found here: :doc:`./apply_patch/index`
 
+Check your file structure
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The file structure should be identical to the original PIM file structure.
+
+.. note::
+
+    To get the original PIM file structure, you can install on an empty directory the PIM from the Portal Enterprise Repository.
```

Comment:
Maybe this part is too much after all, I'm afraid it might confuse customer who will re-download the PIM...